apps/src/code-studio/pd/workshop_dashboard/components/survey_results/matrix_choice_responses.jsx

```diff
@@ -15,14 +15,35 @@ export default class MatrixChoiceResponses extends React.Component {
     questionId: PropTypes.string.isRequired
   };
 
+  // facilitator responses are in form {facilitator1: {question1: <answer>,...}, facilitator2: {...}}
+  // Extract out facilitator answers for a given question id and return in the format
+  // {facilitator1: answer, facilitator2: answer}
+  getFacilitatorAnswers(innerQuestionId) {
+    const {answer} = this.props;
+    let facilitatorAnswers = {};
+    for (const facilitatorId in answer) {
+      if (answer[facilitatorId][innerQuestionId]) {
+        facilitatorAnswers[facilitatorId] =
+          answer[facilitatorId][innerQuestionId];
+      }
+    }
+    return facilitatorAnswers;
+  }
+
   render() {
     const {section, answer, question, questionId} = this.props;
 
     return (
       <div>
         {_.compact(
           Object.keys(question['rows']).map(innerQuestionId => {
-            const innerAnswer = answer[innerQuestionId];
+            let innerAnswer = null;
```

Comment:
Do we define "innerAnswer" in a comment anywhere?

Reply:
will add comment